utils/bigint/src/bigint_ser.rs

```diff
@@ -51,5 +51,5 @@ where
             ));
         }
     };
-    Ok(BigInt::from_bytes_be(sign, &bz))
+    Ok(BigInt::from_bytes_be(sign, &bz[1..]))
```

Comment:
What happens if there's only a sign byte? This will panic i think, but what is the expected behaviour? I would assume 0 or something, since empty bytes -> bigint deserializes to 0?

Reply:
on the 0 case, it is serialized as empty bytes, even if it was a sign byte and nothing else, this would just deserialize as zero. Also the functionality is basically the same as it was before.